crane_plus_examples/src/pick_and_place_tf.cpp

```diff
@@ -155,7 +155,14 @@ class PickAndPlaceTf : public rclcpp::Node
     const double GRIPPER_OFFSET = 0.13;
     double gripper_offset_x = GRIPPER_OFFSET * std::cos(theta_rad);
     double gripper_offset_y = GRIPPER_OFFSET * std::sin(theta_rad);
-    control_arm(x - gripper_offset_x, y - gripper_offset_y, 0.05, 0, 90, theta_deg);
+    if (control_arm(
+        x - gripper_offset_x, y - gripper_offset_y, 0.05, 0, 90,
+        theta_deg) != moveit::core::MoveItErrorCode::SUCCESS)
+    {
+      // 軌道生成に失敗した時はpick_and_placeを中断して待機姿勢に戻る
```

Comment:
今回軌道生成とアーム動作のどちらで失敗してるか区別してないのでこんな表現のほうがいいかもしれません。
```suggestion
      // アーム動作に失敗した時はpick_and_placeを中断して待機姿勢に戻る
```